docs/index.adoc

```diff
@@ -6443,6 +6451,109 @@ class Sub implements Runnable {
 }
 ----
 
+.Kotlin
+[source,kotlin,role="secondary"]
+----
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.apache.logging.log4j.core.config.Configurator
+
+import picocli.CommandLine
+import picocli.CommandLine.Command
+import picocli.CommandLine.Mixin
+import picocli.CommandLine.Model.CommandSpec
+import picocli.CommandLine.Option
+import picocli.CommandLine.ParseResult
+import picocli.CommandLine.RunLast
+import picocli.CommandLine.Spec
```

Comment:
I would prefer to reduce the imports somehow; maybe use star imports? 

```
import picocli.CommandLine.*; //?
```

Too many imports can distract from the main point of the example...

Reply:
I fully agree.
The PR uses star imports now.